src/Runtime/ParentRuntime.php

```diff
@@ -21,7 +21,7 @@
  */
 class ParentRuntime extends BaseParentRuntime
 {
-    public static function createProcess($task, ?int $outputLength = null, ?string $binary = 'php'): Runnable
+    public static function createProcess($task, ?int $outputLength = null, ?string $binary = 'php', ?int $max_input_size = 100000): Runnable
     {
         $runnable = parent::createProcess($task, $outputLength, $binary);
```

Comment:
Please forward max imput arg to parent method.

Reply:
Done